api/src/main/java/com/example/api/model/group/Group.java

```diff
@@ -25,8 +25,5 @@ public class Group {
     @OneToMany(mappedBy = "group")
     @JsonBackReference
     private List<User> users;
-
-    @ManyToOne
-    private AccessDate accessDate;
```

Comment:
Czemu AccessDate nie jest już potrzebny i w sumie do czego on służył?

Reply:
Do niczego nie służył, dlatego nie jest potrzebny. Chodziło o to, że można ograniczyć datę "istnienia" grupy. Gdy minął by dateTo, wtedy nie można by było dołączyć do tej grupy. Feature fajny ale niepotrzebny nam

Reply:
Do niczego nie służył, dlatego nie jest potrzebny. Chodziło o to, że można ograniczyć datę dołączenia do grupy. Gdy minął by dateTo, wtedy nie można by było dołączyć do tej grupy. Feature fajny ale niepotrzebny nam

Reply:
Do niczego nie służył, dlatego nie jest potrzebny. Chodziło o to, że można ograniczyć datę dołączenia do grupy. Gdy minął by dateTo, wtedy nie można by było dołączyć do tej grupy. Feature fajny ale niepotrzebny nam